plugins/processors/gpu/processor.go

```diff
@@ -0,0 +1,194 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package gpu
```

Comment:
`gpu` is a bit too generic. It's very specific to this use case and primarily an attribute filter. Consider something like `gpuattributesprocessor`

Reply:
good call